src/components/views/elements/DeviceVerifyButtons.js

```diff
@@ -65,7 +65,8 @@ export default React.createClass({
                         <ul>
                             <li><label>Device name:</label> <span>{ this.state.device.getDisplayName() }</span></li>
                             <li><label>Device ID:</label>   <span><code>{ this.state.device.deviceId}</code></span></li>
-                            <li><label>Device key:</label>  <span><code><b>{ this.state.device.getFingerprint() }</b></code></span></li>
+                            <li><label>Device fingerprint:</label>  <span><code><b>{ this.state.device.getFingerprint() }</b></code></span></li>
+                            <li><label>Device key:</label>  <span><code><b>{ this.state.device.getIdentityKey() }</b></code></span></li>
```

Comment:
While it may not be useful at the moment, I wish there was somewhere that it was exposed to the user so they could see the other "public key" they trust.  If someone's signing key was used to sign "0" as the streaming key, I would be concerned, and it's just odd that you can't find it in the UI (as far as I can tell).